src/reducers/root-ui.js

```diff
@@ -1,10 +1,22 @@
 import * as actions from './../actions/types'
 import { loadKeyState } from '../utils/localStorage'
 
-const isNightModeEnabled = loadKeyState('isNightModeEnabled') || false
-const isBetaModeEnabled = loadKeyState('isBetaModeEnabled') || false
+const isNightMode = loadKeyState('isNightMode')
+const isNightModeDeprecated = loadKeyState('isNightModeEnabled')
+const isBetaMode = loadKeyState('isBetaMode')
+const isBetaModeDeprecated = loadKeyState('isBetaModeEnabled')
 const isNewsEnabled = loadKeyState('isNewsEnabled') || false
 
+let isNightModeEnabled = isNightMode || false
+if (isNightMode === undefined && isNightModeDeprecated !== undefined) {
+  isNightModeEnabled = isNightModeDeprecated
+}
```

Comment:
We could simplify it to 1 line since `undefined` is a **falsy** value 
```js
const isNightModeEnabled = isNightMode || isNightModeDeprecated
```

Reply:
The problem here, that we don't want to use `isNightModeDeprecated`, if `isNightMode = false`